kinematics_base/include/moveit/kinematics_base/kinematics_base.h

```diff
@@ -496,6 +578,8 @@ class KinematicsBase
   double search_discretization_;
   double default_timeout_;
   std::vector<unsigned int> redundant_joint_indices_;
+  std::map<int,double> redundant_joint_discretization_;
```

Comment:
My guess is that the "search_discretization_" member was used for all of the joints whose indices were listed in the "redundant_joint_indices_" array.